.github/workflows/test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: 'Build internal prerequisites'
         run: yarn workspace @modular-scripts/workspace-resolver build
       - name: Run Tests
-        run: yarn test --coverage
+        run: yarn test --coverage --regex="./packages/*"
```

Comment:
yarn test won't work for us, cause most of our packages aren't actual modular packages